src/drawing/DrawingAnnotations.scss

```diff
@@ -26,3 +26,14 @@
     pointer-events: auto;
     touch-action: none;
 }
+
+.ba-DrawingAnnotations-toolbar {
+    opacity: 1;
+    transition: opacity 200ms ease;
+    pointer-events: auto;
+
+    &.ba-is-faded {
```

Comment:
We usually want to avoid describing the visual treatment in the class name. Could we make the "faded-out" treatment the default and use `ba-is-active`, maybe?